sample.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.27"
+    }
+  }
+  required_version = ">= 0.14.9"
+}
+
+provider "aws" {
+  profile = "default"
+  region  = "us-west-1"
+}
+
+resource "aws_instance" "app_seraver" {
```

Comment:
Issue - Ensure that your AWS application is not deployed within the default Virtual Private Cloud in order to follow security best practices 
 category - Infrastructure Security 
 severity - MEDIUM